lib/login/login.app.js

```diff
@@ -2,6 +2,26 @@ function loadModule() {
     (function () {
         'use strict';
 
+        angular.module('chaise.configure-login', [
```

Comment:
Can you please list all the dependencies in here? Including the ones that `modal.js` and `utils.js` might depend on. So we have a list of modules that should not use `chaiseConfig` during the load.

Reply:
I commented here because this is the first place that I saw this. You might want to do it for all the places that we're using this module (all the apps)